.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
 exclude = .git, __pycache__
-ignore = E302
+ignore = E302, W503, W504
```

Comment:
A similar change is in #18.

Reply:
The reason why we suppress both W503 and W504, is that I think it's an outright bug in `flake8` that either of these warnings trigger on cases where you've intentionally spread code across lines by leaving a parenthesis open. The whole point of doing that is *so that* you can put operators at the beginning or end of a line. Whether the code reads more clearly with the operator at the beginning or end can depend on which operator it is.